package.json

```diff
@@ -10,8 +10,13 @@
     "preview": "vite preview"
   },
   "dependencies": {
+    "@fontsource/dm-sans": "^5.0.12",
```

Comment:
this is not how you're supposed to add fonts to the app. 

[one way to do it properly is this: ](https://www.w3schools.com/css/css_font_google.asp)

[Another one is this](https://www.w3docs.com/snippets/css/how-to-import-google-fonts-in-css-file.html)